configurator/templates/AppServer/conf/axis2/axis2.jinja

```diff
@@ -0,0 +1,724 @@
+         domain. There will not be interference between nodes in different groups.
+        -->
+        {% if domain is defined: %}
+            <parameter name="domain">{{domain}}</parameter>
+        {% else: %}
+             <parameter name="domain">wso2.carbon.domain</parameter>
+        {% endif %}
+        <!-- The multicast address to be used -->
+        <!--<parameter name="mcastAddress">228.0.0.4</parameter>-->
+
+        <!-- The multicast port to be used -->
+        <parameter name="mcastPort">45564</parameter>
+
+        <parameter name="mcastTTL">100</parameter>
+
+        <parameter name="mcastTimeout">60</parameter>
+
+        <!--
+           The IP address of the network interface to which the multicasting has to be bound to.
+           Multicasting would be done using this interface.
+        -->
+        <!--
+            <parameter name="mcastBindAddress">127.0.0.1</parameter>
+        -->
+        <!-- The host name or IP address of this member -->
+
+        {% if localMemberHost is defined: %}
+            <parameter name="localMemberHost">{{ localMemberHost }}</parameter>
+        {% else: %}
+            <parameter name="localMemberHost">127.0.0.1</parameter>
+        {% endif %}
+        <!--
+            The bind adress of this member. The difference between localMemberHost & localMemberBindAddress
+            is that localMemberHost is the one that is advertised by this member, while localMemberBindAddress
+            is the address to which this member is bound to.
+        -->
+        <!--
+        <parameter name="localMemberBindAddress">127.0.0.1</parameter>
+        -->
+
+        <!--
+        The TCP port used by this member. This is the port through which other nodes will
+        contact this member
+         -->
+        {% if localMemberPort is defined: %}
+            <parameter name="localMemberPort">{{localMemberPort}}</parameter>
+        {% else: %}
+            <parameter name="localMemberPort">4000</parameter>
+        {% endif %}
+        <!--
+            The bind port of this member. The difference between localMemberPort & localMemberBindPort
+            is that localMemberPort is the one that is advertised by this member, while localMemberBindPort
+            is the port to which this member is bound to.
+        -->
+        <!--
+        <parameter name="localMemberBindPort">4001</parameter>
+        -->
+
+        <!--
+        Properties specific to this member
+        -->
+        <parameter name="properties">
+            <property name="backendServerURL" value="https://${hostName}:${httpsPort}/services/"/>
+            <property name="mgtConsoleURL" value="https://${hostName}:${httpsPort}/"/>
+            {% if subDomain is defined: %}
+                <property name="subDomain" value="{{subDomain}}"/>
+            {% else : %}
+                 <property name="subDomain" value="worker"/>
+            {% endif %}
+        </parameter>
+

```

Comment:
IMO we can define the defaults values in the config files itself rather than making it in the else statements.